Kickstarter-iOS/Views/Controllers/FacebookConfirmationViewController.swift

```diff
@@ -36,8 +37,16 @@ internal final class FacebookConfirmationViewController: UIViewController,
   internal override func viewDidLoad() {
     super.viewDidLoad()
 
+    self.createAccountButton.addTarget(self, action: #selector(createAccountButtonPressed),
+                                       for: .touchUpInside)
```

Comment:
are these just `IBAction`s that were converted to regular target/selectors?

if so could we remove the `@IBAction` annotation and replace with `@objc`?